docs/source/search-experiments.rst

```diff
@@ -30,13 +32,22 @@ The following identifiers are supported:
 Comparator
 ^^^^^^^^^^
 
-The following comparators are supported:
+Comparators for string attributes:
```

Comment:
What about comparators for tags?

Reply:
`string attributes and tags` sounds good!